ToolChain/Assembler/Parser/test_Parser.py

```diff
@@ -1353,26 +1353,27 @@ def test_assembledFile(self):
         """
 
         try:
-            assembler = Assembler("testAssembledFile.casm", "testout.test")
+            assembler = Assembler("ToolChain/Assembler/Parser/testAssembledFile.casm",
+                                  "ToolChain/Assembler/Parser/testout.test")
         except KeyError as e:
             self.assertEqual(1, 0)
 
         try:
-            file = open("testout.test", mode="rb")
+            file = open("ToolChain/Assembler/Parser/testout.test", mode="rb")
             newFile = file.read()
             file.close()
         except KeyError as e:
             self.assertEqual(1, 0)
 
         try:
-            file = open("LoneBall.test", mode="rb")
+            file = open("ToolChain/Assembler/Parser/LoneBall.test", mode="rb")
             testFile = file.read()
             file.close()
         except KeyError as e:
             self.assertEqual(1, 0)
```

Comment:
I just realized I might've misled you here, my bad. Went and did some research. 

If you can clean this to use `self.fail('reason for failing')`, it would be good. But this is fine.

Use : 
```python3
try:
   # possible exception throwing code
except UnexpectedException as e:
   self.fail('Could perfoemd operation and received an unexpected error: {}'.format(e))
```